packages/apps/graph-client/src/components/error-box/error-box.tsx

```diff
@@ -33,7 +34,7 @@ export const ErrorBox = (props: IErrorBoxProps): JSX.Element => {
   }
 
   return (
-    <Notification intent="negative" icon={<SystemIcon.Close />} role="status">
+    <Notification intent="negative" icon={<MonoClose />} role="status">
```

Comment:
![image](https://github.com/kadena-community/kadena.js/assets/112877112/f37b1937-9016-403f-8dc3-86794d380016)

I am not sure if it was already, but this icon is not aligned. If its outside your scope please ignore this :)